src/vm_interrupts.rs

```diff
@@ -24,6 +25,8 @@ pub type Result<T> = core::result::Result<T, Error>;
 enum BindStatus {
     Binding(CpuId, ImsicFileId),
     Bound(CpuId, ImsicFileId),
+    Rebinding(CpuId, ImsicFileId, CpuId, ImsicFileId),
+    Cloned(CpuId, ImsicFileId, CpuId, ImsicFileId),
```

Comment:
If we're in the `Cloned` state, we've already read out the interrupt state from the previous CPU and the interrupt file is now free to be reused. Thus I don't think it's necessary to hold onto the old CPU / interrupt file in this state.